### PR TITLE
Swope GitOps Guide

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: gitops/gitops_overview.adoc [hunk 1]
```diff
@@ -1,15 +1,15 @@
 [#gitops-overview]
-= GitOps
+= GitOps overview
```
Comment (swopebe) [inline gitops/gitops_overview.adoc:2]: Please review this topic. Go to preview from ... >"view file" and click on the links.

File: gitops/gitops_console.adoc [hunk 2]
```diff
@@ -1,7 +1,8 @@
 [#gitops-console]
 = GitOps console
+//Please review this file.
```
Comment (swopebe) [inline gitops/gitops_console.adoc]: this hidden comment for your review

File: gitops/gitops_push_pull.adoc [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [#gitops-push-pull]
 = Deploying Argo CD with the Push and Pull model 
+//Please review this file.
```
Comment (mikeshng) [inline gitops/gitops_push_pull.adoc]: I would like some push vs pull content from https://docs.google.com/document/d/1ecIAAotLjdM9BNydF-zTdcYr1mKa0xcwbDezTHOt_HI/edit (page 1) to be mentioned here.

The current doc does not mention why a user would choose one or the other.

Comment (swopebe) [reply gitops/gitops_push_pull.adoc]: @mikeshng OK, I can _add_ to this a bit, but for a complete rewrite of push pull (if needed), I am gonna have to ask the we open a new issue for 2.9 for the team.

Comment (swopebe) [reply gitops/gitops_push_pull.adoc]: Is everything else OK?

Comment (mikeshng) [reply gitops/gitops_push_pull.adoc]: Yes, everything else looks ok regarding app.
For policy, I would double check with the policy team. I did take a look and it seems ok to me.

File: applications/app_model.adoc [hunk 4]
```diff
@@ -14,6 +14,7 @@ View the following application resource sections:
 * <<applicationset,ApplicationSet>>
 * <<applicationdoc,Application documentation>>
 
+*Important:* _Channel_ and _Subscription_ model is in maintenance stage, without significant new feature development. *Best practice:* Use the GitOps Operator or Argo CD integration instead of the Channel and Subscription model. Learn more from the link:../gitops/gitops_overview.adoc#gitops-overview[GitOps overview].
```
Comment (swopebe) [inline applications/app_model.adoc:17]: @mikeshng agreed that this closes: https://issues.redhat.com/browse/ACM-6221, so we can close that issue and continue with the larger GitOps review.

**Pull request merged**